src/node/plugin.ts

```diff
@@ -24,14 +24,15 @@ const isPageChunk = (
 
 export function createVitePressPlugin(
   root: string,
-  { configPath, alias, markdown, site }: SiteConfig,
+  { configPath, alias, markdown, site, vueOptions }: SiteConfig,
   ssr = false,
   pageToHashMap?: Record<string, string>
 ): Plugin[] {
   const markdownToVue = createMarkdownToVueRenderFn(root, markdown)
 
   const vuePlugin = createVuePlugin({
-    include: [/\.vue$/, /\.md$/]
+    include: [/\.vue$/, /\.md$/],
+    ...vueOptions
```

Comment:
should have a fallback value here

Reply:
Are you referring to the case where `vueOptions` is `undefined`? It's valid to spread `undefined` so I didn't think a fallback is necessary.

Reply:
Oh what? I thought it's invalid to spread undefined... nvm 😅

Reply:
Ya I think you can also spread `null` too haha! Thanks for the code review 😄